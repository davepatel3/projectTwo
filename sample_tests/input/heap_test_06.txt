1351
get_largest_file_handle
create_directory_root
F2H8OYl4oQ8PAR
create_link_root
1
WZvH
create_directory
Xy Sgb3
1
create_file
7
Mo5f1m0gFOatFe
1
get_largest_file_handle
remove
2
create_directory_root
TgqAsjrURYbLOqz
create_link
1
FkM VS--yAc
1
create_link
5
AWOsXs
1
create_link
4
-iZITdgJ
3
create_file
44
Bu7gScqci7
3
create_link
2
stWATyJAVukcMo2Bn
2
rename
6
CTdQ2f7LyMwntklVB- 
create_file
16
5SIq5mbTcXU1
1
rename
7
uCyidILZ
create_directory
sWn7Lo
2
create_directory
cK1pB1G7m
1
create_directory
KcjE73ULsRx
1
create_file
17
mPeA
3
get_largest_file_handle
create_link
1
h5EjprT
11
rename
3
6BZicZQto6
print_layout
create_directory
aZpe8_hdhy-Hq
11
create_directory
FJawzS
0
get_absolute_path
9
create_file
9
OVOHWIdxQOtJ-oJm 
3
print_layout
rename
15
tbxrDkx7JmQVz-u-4w-
get_largest_file_handle
print_layout
create_file_root
82
084x3fEr2
create_directory
cbg2DKwd-SA
2
create_directory
IpiK
0
create_link
8
aaFbssPSVg1S
12
rename
6
CxHx0Do
remove
11
create_link
15
iV-swuzsa
20
create_link_root
7
5 DCwyEPSF1CujEDJOAD
create_link
7
8f6-moMM4t
1
create_link
20
coFjc32pEyE
11
create_file
6
4r m MfN
2
create_file
15
gWWioADkc
20
create_file
38
iQKftOK
11
get_file_size
8
create_directory
Aq2wWgIrTPzBofjcCzxa
3
create_link
5
b uwNrgH Ez7
1
get_largest_file_handle
get_largest_file_handle
create_link
12
eMw0sxVUye66
30
create_file
5
3KrvzizN
16
create_link
32
O2DLUAKjSS1 xLQ6IcK
0
create_file
31
CO9rfvufKIHNBw2oR7lq
3
get_largest_file_handle
get_largest_file_handle
follow
5
create_directory
ryALoVDM0gMYgB
11
get_file_size
5
create_directory
4hYl Q_VeUBhS
13
get_handle
/F2H8OYl4oQ8PAR/6BZicZQto6/uCyidILZ
create_link
23
y F 6vXT39IA
30
create_directory
TlOPI85
17
create_file
24
_ jpCfhyX pPbYHn
39
create_directory
POm9e8X-gij
1
get_available_size
create_directory
SMGZRsPsmX
20
follow
38
create_directory
-I2t0JcEw4
37
create_file
12
BsbfhTsd748
43
get_largest_file_handle
create_file
4
c44c H7cNz8B2NkcXK
37
get_absolute_path
5
create_directory
Z85rBY
41
create_link
31
Sh9TzAsEEx2D_
0
create_file
11
As7Whmxqk hG7an7rSjL
39
get_available_size
create_directory
pUP2r4JYg1E_c
13
get_file_size
26
create_directory
wUaQk
16
create_file
36
YjF1LNBIV
13
follow
6
quit
