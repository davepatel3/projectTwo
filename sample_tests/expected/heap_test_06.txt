get_largest_file_handle
Heap is empty!
create_directory_root
1
create_link_root
2
create_directory
3
create_file
4
get_largest_file_handle
4
remove
create_directory_root
2
create_link
5
create_link
6
create_link
7
create_file
8
create_link
9
rename
create_file
10
rename
create_directory
11
create_directory
12
create_directory
13
create_file
14
get_largest_file_handle
8
create_link
15
rename
print_layout
[D]F2H8OYl4oQ8PAR
	[D]6BZicZQto6
		[L]uCyidILZ [->/F2H8OYl4oQ8PAR/Mo5f1m0gFOatFe]
		[F]Bu7gScqci7 (size = 44)
		[F]mPeA (size = 17)
	[F]Mo5f1m0gFOatFe (size = 7)
	[L]FkM VS--yAc [->/F2H8OYl4oQ8PAR]
	[L]CTdQ2f7LyMwntklVB-  [->/F2H8OYl4oQ8PAR]
	[F]5SIq5mbTcXU1 (size = 16)
	[D]cK1pB1G7m
	[D]KcjE73ULsRx
[D]TgqAsjrURYbLOqz
	[L]stWATyJAVukcMo2Bn [->/TgqAsjrURYbLOqz]
	[D]sWn7Lo
		[L]h5EjprT [->/F2H8OYl4oQ8PAR]

create_directory
16
create_directory
17
get_absolute_path
/TgqAsjrURYbLOqz/stWATyJAVukcMo2Bn
create_file
18
print_layout
[D]F2H8OYl4oQ8PAR
	[D]6BZicZQto6
		[L]uCyidILZ [->/F2H8OYl4oQ8PAR/Mo5f1m0gFOatFe]
		[F]Bu7gScqci7 (size = 44)
		[F]mPeA (size = 17)
		[F]OVOHWIdxQOtJ-oJm  (size = 9)
	[F]Mo5f1m0gFOatFe (size = 7)
	[L]FkM VS--yAc [->/F2H8OYl4oQ8PAR]
	[L]CTdQ2f7LyMwntklVB-  [->/F2H8OYl4oQ8PAR]
	[F]5SIq5mbTcXU1 (size = 16)
	[D]cK1pB1G7m
	[D]KcjE73ULsRx
[D]TgqAsjrURYbLOqz
	[L]stWATyJAVukcMo2Bn [->/TgqAsjrURYbLOqz]
	[D]sWn7Lo
		[L]h5EjprT [->/F2H8OYl4oQ8PAR]
		[D]aZpe8_hdhy-Hq
[D]FJawzS

rename
get_largest_file_handle
8
print_layout
[D]F2H8OYl4oQ8PAR
	[D]6BZicZQto6
		[L]uCyidILZ [->/F2H8OYl4oQ8PAR/Mo5f1m0gFOatFe]
		[F]Bu7gScqci7 (size = 44)
		[F]mPeA (size = 17)
		[F]OVOHWIdxQOtJ-oJm  (size = 9)
	[F]Mo5f1m0gFOatFe (size = 7)
	[L]FkM VS--yAc [->/F2H8OYl4oQ8PAR]
	[L]CTdQ2f7LyMwntklVB-  [->/F2H8OYl4oQ8PAR]
	[F]5SIq5mbTcXU1 (size = 16)
	[D]cK1pB1G7m
	[D]KcjE73ULsRx
[D]TgqAsjrURYbLOqz
	[L]stWATyJAVukcMo2Bn [->/TgqAsjrURYbLOqz]
	[D]sWn7Lo
		[L]tbxrDkx7JmQVz-u-4w- [->/F2H8OYl4oQ8PAR]
		[D]aZpe8_hdhy-Hq
[D]FJawzS

create_file_root
19
create_directory
20
create_directory
21
create_link
22
rename
remove
create_link
23
create_link_root
24
create_link
25
create_link
26
create_file
27
create_file
28
create_file
29
get_file_size
44
create_directory
30
create_link
31
get_largest_file_handle
19
get_largest_file_handle
19
create_link
32
create_file
33
create_link
34
create_file
35
get_largest_file_handle
19
get_largest_file_handle
19
follow
1
create_directory
36
get_file_size
Invalid handle!
create_directory
37
get_handle
7
create_link
38
create_directory
39
create_file
40
create_directory
41
get_available_size
1057
create_directory
42
follow
1
create_directory
43
create_file
44
get_largest_file_handle
19
create_file
45
get_absolute_path
/F2H8OYl4oQ8PAR/FkM VS--yAc
create_directory
46
create_link
47
create_file
48
get_available_size
1030
create_directory
49
get_file_size
Invalid handle!
create_directory
50
create_file
51
follow
1
quit
